content.css

```diff
@@ -44,7 +44,7 @@
 .show-modal {
   opacity: 1;
   visibility: visible;
-  transform: scale(1.0);
+  transform: scale(1);
```

Comment:
looks like my linter took off the decimal. If you prefer it can stay on, but you would need to add it back probably,

Reply:
Should be fine! I think scale can take both.